.github/workflows/rust.yml

```diff
@@ -16,7 +16,7 @@ jobs:
           toolchain: stable
 
       - name: Run tests
-        run: cargo test --all-targets
+        run: cargo test
```

Comment:
It turns out benchmarks are also ran if we use the `--all-targets` flag, so remove.